Wrappers/Python/cil/processors/PaganinProcessor.py

```diff
@@ -315,7 +293,7 @@ def set_input(self, dataset):
         return super().set_input(dataset)
 
     def get_output(self, out=None, override_geometry=None,
-                   override_filter=None):
+                   override_filter=None, verbose=True):
```

Comment:
This seems overkill just for tqdm viability. Do we need to be able to hide it? Could we use logging `instead`? Should it be a general argument for other processors?

Reply:
I've updated this so I'm just suppressing the output in the tests, so I've removed the verbose argument